packages/core-scroll/core-scroll.js

```diff
@@ -77,10 +77,22 @@ export default class CoreScroll extends HTMLElement {
     window.addEventListener('load', this) // Update state when we are sure all CSS is loaded
     document.addEventListener('click', this)
     setTimeout(() => this.handleEvent()) // Initialize buttons after children is parsed
+    
+    // Initialize buttons when children change
+    // - jsx relies on onScrollChange triggering to update button states
+    this._childListObserver = new MutationObserver((mutationList) => {
```

Comment:
Ref similar implementation in [core-tabs](https://github.com/nrkno/core-components/blob/master/packages/core-tabs/core-tabs.js#LL17). I propose to add the following:
* Make sure MutationObserver exists on `window`, and call using `window.MutationObserver` to avoid `undefined` issues
* Move function to be called out to a standalone function next to CoreScroll (example [here](https://github.com/nrkno/core-components/blob/master/packages/core-tabs/core-tabs.js#L144))

Reply:
great ideas! I fix 🧑‍🏭